sdk/python/kfp/dsl/for_loop.py

```diff
@@ -99,25 +125,23 @@ def __init__(
             name_code: A unique code used to identify these loop arguments.
                 Should match the code for the ParallelFor ops_group which created
                 these LoopArguments. This prevents parameter name collisions.
-            name_override: The override name for PipelineParameterChannel.
-            **kwargs: Any other keyword arguments passed down to PipelineParameterChannel.
+            name_override: The override name for PipelineChannel.
+            **kwargs: Any other keyword arguments passed down to PipelineChannel.
```

Comment:
(Not required for this PR)

AFAICT, since this `**kwargs` is an implementation detail and we know all consumers of this API, there's no reason to have `**kwargs`. It would be nice to remove this by converting to explicit arguments at some point.